lib/reducers/middleware.js

```diff
@@ -0,0 +1,14 @@
+import { clearAllWallets } from '../actions';
```

Comment:
I'm not sure middleware makes sense in the reducers folder. Middleware is kind of its own thing in redux. In bpanel we put it in `webapp/store`, which includes a directory for `reducers` and a `middleware.js` file. Thoughts? I do like having it split out though and keeping `index.js` really clean.

Reply:
I think that making a whole new directory is a lot but if you are really opinionated then we can do that, otherwise I think having it in the reducers isn't that big of a deal because its called `middleware` which gets ran before the reducers

Reply:
Yeah, I don't think a whole new directory is necessary, more just thinking it could be separate in parallel to `reducers`.